src/Components/Members/Table/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styles from './table.module.css';
+
+const Table = ({ data, deleteMemb, setUpd, showUpd, setSelectId }) => {
+  return (
+    <table className={styles.table}>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Last Name</th>
+          <th>Dni</th>
+          <th>Phone</th>
+          <th>Email</th>
+          <th>City</th>
+          <th>Birthdate</th>
+          <th>Zip</th>
+          <th>Is active?</th>
+          <th>Membership type</th>
+          <th>Tools</th>
+        </tr>
+      </thead>
+      <tbody>
+        {data.map((i) => {
```

Comment:
Let's give a more descriptive name to the iterator, we could use `item` instead of `i`
```suggestion
        {data.map((item) => {
```

Reply:
Ready